docs/pages/includes/plugins/config-toml-teleport.mdx

```diff
@@ -14,8 +14,9 @@ are not using them in this configuration.
 </TabItem>
 <TabItem label="Helm Chart">
 
-**`address`**: Include the hostname and HTTPS port of your Teleport Cloud tenant
-(e.g., `teleport.example.com:443`).
+**`address`**: Include the hostname and HTTPS port of your Teleport Proxy Service
+or Teleport Cloud tenant (e.g., `teleport.example.com:443` or
```

Comment:
The Core Concepts page says "Teleport Enterprise Cloud" instead of "Teleport Cloud"

Reply:
updated